test/integration/homepage-rows.test.js

```diff
@@ -0,0 +1,59 @@
+const SeleniumHelper = require('./selenium-helpers.js');
+
+const {
+    clickXpath,
+    findByXpath,
+    buildDriver
+} = new SeleniumHelper();
+
+let remote = process.env.SMOKE_REMOTE || false;
+let rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
+
+if (remote) {
+    jest.setTimeout(60000);
+} else {
+    jest.setTimeout(20000);
+}
+
+let driver;
+
+describe('www-integration project rows', () => {
+    beforeAll(async () => {
+        driver = await buildDriver('www-integration project rows');
+        // driver.get(rootUrl);
+    });
+
+    beforeEach(async () => {
+        await driver.get(rootUrl);
+    });
+
+    afterAll(async () => await driver.quit());
+
+    test('Featured Projects row title', async () => {
+        let projects = await findByXpath('//div[@class="box"]/div[@class="box-header"]/h4');
```

Comment:
As a follow-up, I think it would be justifiable to add classes to find these, since this is fragile.